app/models/product.rb

```diff
@@ -22,8 +15,14 @@ class Product < ApplicationRecord
     message: 'must be a URL for GIF, JPG or PNG image.'
   }
   validates :title, length: {minimum: 10}
-  validates :price, numericality: { greater_than_or_equal_to: 0.01 }
+  validates :price, numericality: { greater_than_or_equal_to: 0.01 }, allow_blank: true
```

Comment:
@maskedXchaotic I think we should remove this validation:  `validates :price, numericality: { greater_than_or_equal_to: 0.01 }, allow_blank: true`. We already have another validation where we are checking that price should be greater than the discounted price. 

And add validation for discounted price, that it should be present and numeric